de-lib/src/main/java/org/iplantc/de/diskResource/client/events/selection/MetaTemplateInfoBtnSelected.java

```diff
@@ -0,0 +1,42 @@
+package org.iplantc.de.diskResource.client.events.selection;
+
+import org.iplantc.de.client.models.diskResources.MetadataTemplateInfo;
+
+import com.google.gwt.event.shared.EventHandler;
+import com.google.gwt.event.shared.GwtEvent;
+import com.google.gwt.event.shared.HandlerRegistration;
+
+/**
+ * A GWT event that fires when the user hits the "info" button for a metadata template
+ */
+public class MetaTemplateInfoBtnSelected
+        extends GwtEvent<MetaTemplateInfoBtnSelected.MetaTemplateInfoBtnSelectedHandler> {
+    public static interface MetaTemplateInfoBtnSelectedHandler extends EventHandler {
```

Comment:
Aren't all interfaces static?

Reply:
Yes - I use IntelliJ to automatically create these GWT events for me and it adds in `static` for some reason.  I'll clean those up.

Reply:
There's no need to.  I shouldn't have asked that question here.  I'm sorry about that.

Reply:
No worries, it's a valid question!